### DISCUSSION TITLE
希望应用图标可以改进一下

### DISCUSSION THREAD

**Discussion by 123Duo3** [Richasy/Bili.Uwp#74] [想法]
现有的图标似乎不是很能融入到系统应用的图标中，而且因为层级太多加上阴影导致在比较小的缩放下看起来有点脏（？
而且中间的白色的渐变看起来很奇怪，有种光面塑料的感觉而不是稍微有点磨砂的亚克力_(:з」∠)_
所以希望换一个来搭配出色的界面(｀・ω・´）
![%`F~4P)W`}237CV8T7 E$IO](https://user-images.githubusercontent.com/12569997/133892512-d8377c40-6593-40f8-92ce-f18d19295299.png)
（自己画了个脑子里的草图您看看行不行）

**Comment by Richasy** [comment]
暂时没有更改图标的打算哟，我先把bug修掉
